migrations/versions/rev-2023-06-15-08:53:12-d3c32d203ef1_.py

```diff
@@ -0,0 +1,31 @@
+"""empty message
+Revision ID: d3c32d203ef1
+Revises: a45167cdc102
+Create Date: 2023-06-15 08:53:12.741108
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd3c32d203ef1'
+down_revision = 'a45167cdc102'
```

Comment:
I am not sure this is the correct down revision, since there is
```
rev-2022-04-04-17:35:59-a45167cdc102_.py
rev-2023-06-12-13:39:49-410e54b84481_.py
```
so I guess it should be `410e54b84481`

Reply:
It's has been resolved.